object-manipulation-1/src/get-values.js

```diff
@@ -0,0 +1,5 @@
+/* eslint-disable no-unused-vars */
+function getValues(object) {
+  const values = Object.values(object);
```

Comment:
Please do not use `Object.values` for this.

Reply:
Sorry for missed instruction on this, I will revising it now.

Reply:
Issue resolved with new commit 
ab67dc5